src/main/java/io/confluent/connect/jdbc/source/TableMonitorThread.java

```diff
@@ -105,6 +124,10 @@ public void run() {
     if (tables == null) {
       throw new ConnectException("Tables could not be updated quickly enough.");
     }
+    if (!duplicates.isEmpty()) {
+      log.info("The filtered tables are {} ", tables);
+      throw new ConnectException(configErrorMsg + duplicates);
```

Comment:
What do you think about printing the pairs of TableIds (e.g., `a.b.Foo / a.c.Foo`) rather than just dumping the map. I'm worried that people won't know what the map output means at the end of the long error message.

Reply:
could use duplicates.values()